README.md

```diff
@@ -20,3 +20,75 @@ with a few commands. Need an ingress to serve traffic? Use Juju to deploy and
 integrate a range of ingresses, such as
 [Traefik](https://charmhub.io/traefik-k8s), and expose your application to
 external traffic in seconds.
+
+## Contributing
+
+Is there something missing, for example, a framework that you would like to use?
+PaaS App Charmer welcomes contributions! This section covers how to add a new
+integration and a new framework.
+
+### Add an Integration
+
+There are a few recommended steps to add a new integration which we'll go
+through below.
+
+1. Please write a proposal on the
+  [charm](https://discourse.charmhub.io/c/charm/41) topic on discourse. This
```

Comment:
I think the link should be for "charm topic on discourse" not just "charm", otherwise it seems like this could be a link to what a charm is.